sdk/common/src/test/java/io/opentelemetry/sdk/internal/ThrowableUtilTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.internal;
+
+import static io.opentelemetry.sdk.internal.ThrowableUtil.propagateIfFatal;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.Test;
+
+class ThrowableUtilTest {
+
+  @Test
+  void propagateIfFatal_Throws() {
+    assertThatCode(() -> propagateIfFatal(new RuntimeException("Error!")))
+        .doesNotThrowAnyException();
+    assertThatCode(() -> propagateIfFatal(new Exception("Error!"))).doesNotThrowAnyException();
+    assertThatCode(() -> propagateIfFatal(new VirtualMachineError("Error!") {}))
```

Comment:
Let's use `assertByThrownBy` for readability for the throwing ones. It's interesting how assertj seems to have just aliased these two